lib/text/ui_text_displayer.js

```diff
@@ -247,7 +247,9 @@ shaka.text.UITextDisplayer = class {
     // Set white-space to 'pre-line' to enable showing line breaks in the text.
     captionsStyle.whiteSpace = 'pre-line';
     captions.textContent = cue.payload;
-    captionsStyle.backgroundColor = cue.backgroundColor;
+    if (isNested) {
```

Comment:
If the parent cue doesn't have nested cues, shall we still apply the background color to the parent cue?

Reply:
I have never seen that case, but if you suggest a better fix, I'll apply it!

Reply:
if (!cue.nestedCues.length || isNested) {
    // If the cue is a parent cue without nested cues, apply the background color.
    // Or, if the cue is a nested cue, apply the background color.
    captionsStyle.backgroundColor = cue.backgroundColor;
}
Does that make any sense to you?

Reply:
The `displayCue_` function is only called from `updateCaptions_`. The function takes into account whether the `cue` has `nestedCues`. And the `isNested` parameter is populated based on this, I think `isNested` is enough.

Reply:
I see. Thanks!